gobblin-metastore/src/main/java/org/apache/gobblin/metastore/DatasetStateStore.java

```diff
@@ -56,7 +56,7 @@
   public void persistDatasetURNs(String storeName, Collection<String> datasetUrns) throws IOException;
 
   @Override
-  default List<? extends DatasetStateStoreEntryManager> getMetadataForTables(StateStorePredicate predicate)
+  default List<? extends DatasetStateStoreEntryManager> getEntryManagersForTables(StateStorePredicate predicate)
```

Comment:
I would prefer if this was not renamed, as it is a public method. Unless it is strictly necessary.